purview/app.py

```diff
@@ -15,6 +15,11 @@
 
 
 def load_config(app):
+    """
+    Determine the correct configuration to use and apply it.
```

Comment:
You can remove the "and apply it" part if you want. It seems like extra

Reply:
I don't think it's implied by the function name so I'll keep it.